### PR TITLE
Add cluster status skipped

### PR DESCRIPTION
**Pull request by rJankowski93**
https://github.com/kyma-incubator/reconciler/issues/277

### CODE REVIEW

File: pkg/cluster/inventory.go [hunk 1]
```diff
@@ -187,6 +188,44 @@ func (i *DefaultInventory) createStatus(configEntity *model.ClusterConfiguration
...
 	}
 
+	// create new reconcile pending statuses
+	if status == model.ClusterStatusReconcilePending {
+		q, err := db.NewQuery(i.Conn, &model.ClusterStatusEntity{}, i.Logger)
+		if err != nil {
+			return nil, err
+		}
+
+		notReconciledStatues, err := q.Select().
```
Comment (tobiscr) [inline pkg/cluster/inventory.go:198]: We should not always query the whole history.  Would propose following flow:

* Implement DB migration logic (will require #442 ) and migrate all ol pending-status which were not applied to `skipped`
* Check just the previous cluster-state status, and if this is `pending`, just change this one to `skipped`

**Review by tobiscr — CHANGES_REQUESTED** [review]

**Comment by nachtmaar** [comment]
@tobiscr Can we close this PR ?

**Comment by tobiscr** [comment]
Caused by strategy change to change the reconciler architecture to K8s native app, this feature isn't required anymore

**Pull request closed without merging**